packages/dds/tree/src/shared-tree-core/branch.ts

```diff
@@ -393,51 +403,66 @@ export class SharedTreeBranch<TEditor extends ChangeFamilyEditor, TChange> exten
 	 * This is applicable when a commit is replaced by a rebase or a local commit is sequenced.
 	 */
 	public updateRevertibleCommit(commit: GraphCommit<TChange>) {
-		if (this._revertibleCommits.get(commit.revision) !== undefined) {
+		const revertibleCommit = this._revertibleCommits.get(commit.revision);
```

Comment:
you extracted this into a variable but then didn't use it in a new way. was there an intention that was missed?

Reply:
nah, just like 5 iterations of changes :)